.github/workflows/ci-synthetic-test-runner.yml

```
@@ -0,0 +1,150 @@
+    steps:
+      - name: Set dynamic commit status context
+        id: metadata
+        run: echo "RUN_CONTEXT=synthetic-test-run-$(date -u +'%Y-%m-%dT%H-%M-%S')" >> $GITHUB_OUTPUT
+
+      - name: Verify checkbox is still checked
+        uses: actions/github-script@v7
+        with:
+          script: |
+            console.log('PR_NUMBER from env:', process.env.PR_NUMBER);
+            console.log('PR_NUMBER type:', typeof process.env.PR_NUMBER);
+            
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)